zebra/debug_nl.c

```diff
@@ -1730,6 +1750,21 @@ void nl_dump(void *msg, size_t msglen)
 		nlncm_dump(ncm, nlmsg->nlmsg_len - NLMSG_LENGTH(sizeof(*ncm)));
 		break;
 
+	case RTM_NEWQDISC:
+	case RTM_DELQDISC:
+	case RTM_NEWTCLASS:
+	case RTM_DELTCLASS:
+	case RTM_NEWTFILTER:
+	case RTM_DELTFILTER:
+		tcm = NLMSG_DATA(nlmsg);
+		zlog_debug(
```

Comment:
Can you post some output to show what this looks like when debugging?

Reply:
2022/08/05 14:39:49 ZEBRA: [PZNF6-7DWEC]  tcm [type=qdisc family=AF_UNSPEC (0) ifindex=7 handle=1007:0000]
2022/08/05 14:39:49 ZEBRA: [PZNF6-7DWEC]  tcm [type=tclass family=AF_UNSPEC (0) ifindex=7 handle=1007:0001]
2022/08/05 14:39:49 ZEBRA: [PZNF6-7DWEC]  tcm [type=tfilter family=AF_UNSPEC (0) ifindex=7 handle=0000:0001]